UnitsNet.Tests/CustomCode/SpeedTests.cs

```diff
@@ -68,6 +68,12 @@ public class SpeedTests : SpeedTestsBase
 
         protected override double MillimetersPerHourInOneMeterPerSecond => 3600000;
 
+        protected override double InchesPerSecondInOneMeterPerSecond => 39.3700787;
```

Comment:
Oh yeah, I might as well ask you how should I have done the two pieces of work?
Should I have created two forks? one for the bugfix and one for the addfeature?